pandas/tests/groupby/test_groupby.py

```diff
@@ -145,6 +145,13 @@ def test_len_nan_group():
     assert len(df.groupby(["a", "b"])) == 0
 
 
+def test_groupby_timedelta_median():
+    # issue 57926
+    df = DataFrame({"label": ["foo", "foo"], "timedelta": [pd.NaT, Timedelta("1d")]})
+    median = df.groupby("label")["timedelta"].median()
+    assert median.loc["foo"] == Timedelta("1d")
```

Comment:
Can you check the entire result - this line should be:

    tm.assert_series_equal(result, expected)

where you hardcode the construction of `expected`.